### ISSUE TITLE
[BUG] [Python] Indentation error 

### ISSUE THREAD

**Issue by remy2wescale** [stateful/runme#386]
Hello, 

I would like to use Python integration of runme for document [doit](https://pydoit.org/contents.html) code but i have alway indentation error when i run the README (the same code works well outside runme).

Any ideas ?
 
```
python { interpreter=python3 shell=python3 }
DOIT_CONFIG = {
    'verbosity': 2,
    'minversion': '0.24.0',
    'default_tasks': [
        't3',
    ]
}

def task_t3():
    return {'actions': ["echo task3"]}
```
```
runme run --chdir="/home/noiretr/WORKS/python-doit" --filename="RUN-README.md" --index=0
  File "/home/noiretr/WORKS/python-doit/.runme-script-ckb9s45re1f7ll06qoh0.py", line 10
    return {'actions': ["echo task3"]}
    ^
IndentationError: expected an indented block after function definition on line 9
exit code: 1

**Comment by pastuxso** [comment]
Hi @remy2wescale, the VSCode extension is functioning properly. However, we need to investigate what's happening in the CLI.

**Comment by pastuxso** [comment]
@remy2wescale, this issue has been resolved in runme v1.7.8

**Comment by remy2wescale** [comment]
Yes, i confirm the issue is now resolved.

Thank you for your support
